gtk/src/application.rs

```diff
@@ -30,21 +28,4 @@ impl Application {
             *signalid.borrow_mut() = Some(id);
         }
     }
-
-    #[doc(alias = "gtk_application_new")]
-    pub fn new(
-        application_id: Option<&str>,
-        flags: ApplicationFlags,
-    ) -> Result<Application, glib::BoolError> {
-        skip_assert_initialized!();
-        let app: Application = unsafe {
-            Option::from_glib_full(ffi::gtk_application_new(
-                application_id.to_glib_none().0,
-                flags.to_glib(),
-            ))
-            .ok_or_else(|| glib::bool_error!("Failed to create application"))?
-        };
-        Application::register_startup_hook(&app);
```

Comment:
This is necessary so that GTK is considered initialized or not?

Reply:
I think it's only necessary for the Builder because we are not using the constructor there.

Reply:
Same comment here as for the gtk4 version :)